compiler/src/iree/compiler/Codegen/Common/PolynomialApproximationPass.cpp

```diff
@@ -62,7 +56,7 @@ class PolynomialApproximationPass final
 
     for (const auto &[fnName, populateFn] : patternMap) {
       // Skip any ops in the "do not convert" list.
-      if (!llvm::is_contained(noApproxOps, fnName)) {
+      if (!llvm::is_contained(noApproxOps, fnName) && !clNativeMathPrecision) {
```

Comment:
FYI, this is not the original meaning of the 'native math precision' in this flag name. This is used in a bunch of scripts already, so we may want to pick a new flag for this to avoid confusion.